job-service-container/src/test/java/com/hpe/caf/services/job/api/JobServiceIT.java

```diff
@@ -1129,6 +1142,57 @@ public void testInvalidLabelFormat() {
         }
         assertTrue(exceptionThrown);
     }
+    
+    @Test
+    public void testDeleteLog() throws SQLException
+    {
+        //prepare
+        List<String> droppedTables = new ArrayList();
+        try (final java.sql.Connection dbConnection = getDbConnection();
+        ) {
+            int totalCount = 100;
```

Comment:
@dermot-hardy Although I have tested with a count of 30,000 tables with this test, I am checking in a smaller number. I can add a bigger number here if we are interested in this test to run with a bigger number at every check-in.

Reply:
I would have thought that it should be a larger number than was possible before but I don't know what the cost is.  If it takes significantly longer then 100 is fine.

Reply:
![image](https://user-images.githubusercontent.com/4994904/112615498-909e1200-8e48-11eb-895e-552722ab4c5b.png)

Above screenshot is from jenkins [job](http://sou-jenkins2.swinfra.net/job/JobService/view/Developer/job/JobService~job-service~SCMOD-10825~CI/51). The creation of 30k tables takes approx 5 minutes. Dropping those tables is taking 75 seconds. I checked these numbers for two runs. So, keeping a count of 30k would increase the build time by ~ 6 minutes.

Reply:
Ok do the full check for the release build and the quick check for the CI build.

Reply:
Made changes to exclude the long running IT from CI builds and to include it in release builds only.